apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/UpdateHistory.scala

```diff
@@ -1856,6 +1864,9 @@ class UpdateHistory(
             case Some((updatesComplete, importUpdatesComplete)) =>
               if (updatesComplete && importUpdatesComplete) {
                 BackfillingState.Complete
+              } else if (updatesComplete && !enableImportUpdateBackfill) {
+                // If import update backfilling is disabled, behave as if it was not implemented
```

Comment:
Change 2: if import updates backfilling is disabled, consider the whole backfilling as complete if the regular update backfilling completed. This will prevent the trigger from attempting to backfill import updates, and the UI from complaining that backfilling hasn't completed.